src/table/actionHeaderFactory.js

```diff
@@ -86,34 +99,111 @@ function actionColumnFrom(
     return {};
   }
 
+  let inheritActionAmmo = false;
+  let selectedRow = false;
+
+  const handleInheritClick = (
+    rowIndex,
+    formData,
+    onChange,
+    inheritedAction
+  ) => {
+    const handleOutsideClick = e => {
+      inheritActionAmmo = false;
+      selectedRow = false;
+    };
+    document.addEventListener("click", handleOutsideClick, false);
```

Comment:
When is this event listener getting removed?

Reply:
Added remove event listener